app/pages/reservation/reservation-confirmation/ReservationConfirmation.js

```diff
@@ -18,24 +21,28 @@ class ReservationConfirmation extends Component {
     resource: PropTypes.object.isRequired,
     t: PropTypes.func.isRequired,
     user: PropTypes.object.isRequired,
-    history: PropTypes.object.isRequired,
   };
 
-  handleReservationsButton() {
-    this.props.history.replace('/my-reservations');
+  state = {
+    reservationPrice: null,
+  }
+
+  componentDidMount() {
+    const { reservation, resource } = this.props;
+    if (hasProducts(resource)) {
+      getReservationPrice(apiClient, reservation.begin, reservation.end, resource.products)
+        .then(reservationPrice => this.setState({ reservationPrice }));
```

Comment:
Unlike other components, `catch` is missing here.

Reply:
This is in `componentDidMount` so `state.reservationPrice` is already null here. Maybe setting an error message would be the appropriate thing to do in `catch`?